backend/src/openbeheer/zaaktype/api/views.py

```diff
@@ -481,6 +483,30 @@ def format_version(self, data: ZaakType) -> VersionSummary:
             concept=data.concept,
         )
 
+    def get_fields(
+        self,
+        data: ExpandableZaakType,
+        option_overrides: Mapping[str, list[OBOption]] = {},
+        *,
+        base_editable: Callable[[str], bool] = bool,
+    ) -> Iterable[OBField]:
+        selectielijst_options = (
+            fetch_resultaat_options(procestype_url=data.selectielijst_procestype)
+            if data.selectielijst_procestype
+            else []
+        )
+
+        expansions = set(map(camelize, self.expansions))
+
+        return super().get_fields(
+            data,
+            option_overrides={"selectielijstklasse": selectielijst_options},
```

Comment:
This is an option override for a nested field 🤔 
Maybe it should be more evident?